### PR TITLE
Fixed conversion to decimal

### PR DESCRIPTION
**Pull request by starkmsu**

### CODE REVIEW

**Review by PaterSantyago — DISMISSED** [review]

**Review by PaterSantyago — APPROVED** [review]

**Review by samodovdi — APPROVED** [review]

File: src/Lykke.Numerics/Money.Conversions.cs [hunk 1]
```diff
@@ -78,9 +77,26 @@ public partial struct Money
 
         public static explicit operator decimal(Money value)
         {
-            return (decimal) value._significand / (decimal) Pow10(value._scale);
+            var decimalMaxValue = new BigInteger(decimal.MaxValue);
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Numerics/Money.Conversions.cs:80]: Add tests for the edge cases
